src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -187,6 +215,8 @@ def __init__(self, *args):
             thread_name = s['sm_name']
 
             if not s['sm_enable']:
+                # XXX
```

Comment:
What dos XXX mean? ;)

Reply:
Old convention marking dirty hacks or code, that may require fix in the future. Cleaning up here is kind of a dirty hack really, but does the job.